main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"], defaults={
+                "description": race_data.get("description", "")
+            }
+        )
+
+        for skill_data in race_data.get("skills", []):
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                race=race,
+                defaults={"bonus": skill_data["bonus"]}
+            )
+
+        guild = None
+        if player_data.get("guild"):
+            guild_data = player_data["guild"]
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={
+                    "description": guild_data.get("description", None)
```

Comment:
The 'description' field in the Guild model is defined to allow null values. Therefore, using 'None' as a default is correct, but ensure that the JSON data can handle 'None' appropriately. If the JSON data might not include a 'description', using an empty string '' could be more consistent with the Race model's handling of descriptions.